site/releases/2024-sep-09/release-notes.qmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "September 9, 2024"
+filters:
+  - tachyons
+---
+
+## Release highlights
+
+We've expanded functionality in the {{< var vm.platform >}}, including a more extensible version of risk areas that allows you to customize guidelines associated with your validation templates, as well as the ability to reset your model workflows.
+
+
+<!---
+PR #906: Added new page to manage Risk Areas and guidelines
+URL: https://github.com/validmind/frontend/pull/906
+Labels: enhancement
+--->
+
+### Manage validation guidelines
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-50-ns}
+We've expanded on our risk area functionality and added a new page to your Workplace Settings: **Risk Areas & Validation Guidelines**.
+
+This replaces the old Risk Areas set up under **Organization**, and allows you to customize guidelines associated with your model validation templates.
+
+###### Documentation
```

Comment:
Global: I don't think these headings add anything, they feel like a label for the button and the alignment is sometimes off off. I'd either remove or move into the button text itself? 

<img width="380" alt="image" src="https://github.com/user-attachments/assets/68b2935f-6755-4b12-8979-a77b3bdacabf">

Reply:
I like these headings as it gives some sort of lead-in, and I can adjust the alignment (the button came indented). My preference is actually not to use the buttons as they are sort of a rendering nightmare, but they do look snazzy.

Reply:
> My preference is actually not to use the buttons as they are sort of a rendering nightmare, but they do look snazzy.

I'd agree on limiting the use for these buttons — as mentioned elsewhere, I originally added them to draw attention to our curated highlights ONLY. They work much better when used sparingly, otherwise our release notes just look like they have 'button pimples'. 😛